webapp/chart.js

```diff
@@ -799,7 +800,23 @@ function chart_init(graph, wnd, affichwnd, with_interval, with_interval2, show_i
                 points_list[index].enqueue([line_id, x, y]);
             }else{
                 var values = get_points_for_data(data, data[8], data[10]);
-                points_list[index].enqueue([line_id, values]);
+                cmatview_counter++; // to count lines from log
```

Comment:
```suggestion
                cmatview_counter[index]++; // to count lines from log
```
only increase for that index.